src/mocks.js

```diff
@@ -3,8 +3,8 @@ const makeServiceWorkerEnv = require('service-worker-mock')
 const HASH = '123HASHBROWN'
 
 export const getEvent = request => {
-  const waitUntil = callback => {
-    callback
+  const waitUntil = async callback => {
+    await callback
```

Comment:
should this call the callback? like 
```suggestion
    await callback()
```
?

Reply:
no i cray.